Fixing ClassNotFoundError due to log4j update

bdhandap opened the pull request:
Fixing ClassNotFoundError due to log4j update

**What?**
* Fixing ClassNotFoundError due to log4j update
* Parsing only video frames for Rekognition example
* Updating JNI version

**Why?**
* log4j version update caused Rekognition lambda sample to fail with
  class not found error.
* Passing audio-video file will crash the lambda sample as audio frames
  cannot be used for bounding boxes.
* KVS Producer Java SDK is also updated in earlier commits which require
  newer version of JNI

unicornss reviewed (commented):
don't we need  log4j2 dependency? 
https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-log4j2

bdhandap commented:
> don't we need log4j2 dependency? https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-log4j2

We do need it. I have moved the dependency below to line number 129. I have removed other log dependencies like slf4j as it's not required and creating class conflict

unicornss commented on src/main/java/com/amazonaws/kinesisvideo/parser/examples/lambda/H264FrameProcessor.java:
```diff
@@ -132,38 +132,44 @@ public static H264FrameProcessor create(final AWSCredentialsProvider credentials
+
+                // Decode H264 frame
+                final BufferedImage decodedFrame = h264Decoder.decodeH264Frame(frame, trackMetadata);
+                log.debug("Decoded frame : {} with timecode : {} and fragment metadata : {}",
```
/nit frame Number :{}

unicornss replied:
should we consider excluding the log4j dependency from   
         <dependency>
            <artifactId>aws-java-sdk-kinesis</artifactId>
            <groupId>com.amazonaws</groupId>
        </dependency>

 through https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/blob/master/pom.xml

bdhandap replied:
We will need the log4j dependency in Java SDK

unicornss commented:
```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ amazon-kinesis-video-streams-parser-library ---
[WARNING] The artifact xml-apis:xml-apis:jar:2.0.2 has been relocated to xml-apis:xml-apis:jar:1.0.b2
[INFO] com.amazonaws:amazon-kinesis-video-streams-parser-library:jar:1.2.5
[INFO] +- com.amazonaws:aws-java-sdk-rekognition:jar:1.12.162:compile
[INFO] |  +- com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile
[INFO] |  |  +- (commons-logging:commons-logging:jar:1.1.3:compile - omitted for conflict with 1.2)
[INFO] |  |  +- commons-codec:commons-codec:jar:1.15:compile
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.13:compile
[INFO] |  |  |  +- (org.apache.httpcomponents:httpcore:jar:4.4.13:compile - omitted for conflict with 4.4.15)
[INFO] |  |  |  +- (commons-logging:commons-logging:jar:1.2:compile - omitted for conflict with 1.1.3)
[INFO] |  |  |  \- (commons-codec:commons-codec:jar:1.11:compile - omitted for conflict with 1.15)
[INFO] |  |  +- software.amazon.ion:ion-java:jar:1.0.2:compile
[INFO] |  |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile (version managed from 2.12.6)
[INFO] |  |  |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.12.3:compile
[INFO] |  |  |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.12.3:compile - omitted for conflict with 2.13.0)
[INFO] |  |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.12.3:compile (version managed from 2.12.6)
[INFO] |  |  |  +- (com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile - version managed from 2.12.6; omitted for duplicate)
[INFO] |  |  |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.12.3:compile - omitted for duplicate)
[INFO] |  |  \- (joda-time:joda-time:jar:2.8.1:compile - omitted for conflict with 2.6)
[INFO] |  \- com.amazonaws:jmespath-java:jar:1.12.162:compile
[INFO] |     \- (com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile - version managed from 2.12.6; omitted for duplicate)
[INFO] +- com.amazonaws:aws-java-sdk-kinesisvideo:jar:1.12.162:compile
[INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  +- io.netty:netty-codec-http:jar:4.1.72.Final:compile
[INFO] |  |  +- io.netty:netty-common:jar:4.1.72.Final:compile
[INFO] |  |  +- io.netty:netty-buffer:jar:4.1.72.Final:compile
[INFO] |  |  |  \- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- io.netty:netty-transport:jar:4.1.72.Final:compile
[INFO] |  |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  |  \- (io.netty:netty-resolver:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- io.netty:netty-codec:jar:4.1.72.Final:compile
[INFO] |  |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  |  \- (io.netty:netty-transport:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  \- (io.netty:netty-handler:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  +- io.netty:netty-handler:jar:4.1.72.Final:compile
[INFO] |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- io.netty:netty-resolver:jar:4.1.72.Final:compile
[INFO] |  |  |  \- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- (io.netty:netty-transport:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  +- (io.netty:netty-codec:jar:4.1.72.Final:compile - omitted for duplicate)
[INFO] |  |  \- io.netty:netty-tcnative-classes:jar:2.0.46.Final:compile
[INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] +- com.amazonaws:aws-java-sdk-kinesis:jar:1.12.162:compile
[INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] +- com.amazonaws:aws-java-sdk-dynamodb:jar:1.12.162:compile
[INFO] |  +- com.amazonaws:aws-java-sdk-s3:jar:1.12.162:compile
[INFO] |  |  +- com.amazonaws:aws-java-sdk-kms:jar:1.12.162:compile
[INFO] |  |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] +- com.amazonaws:amazon-kinesis-video-streams-producer-sdk-java:jar:1.12.0:compile
[INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.11.602; omitted for duplicate)
[INFO] |  +- (com.amazonaws:aws-java-sdk-kinesisvideo:jar:1.12.162:compile - version managed from 1.11.602; omitted for duplicate)
[INFO] |  +- com.google.inject:guice:jar:5.1.0:compile
[INFO] |  |  +- javax.inject:javax.inject:jar:1:compile
[INFO] |  |  +- aopalliance:aopalliance:jar:1.0:compile
[INFO] |  |  \- (com.google.guava:guava:jar:30.1-jre:compile - omitted for conflict with 31.0.1-jre)
[INFO] |  +- com.google.code.findbugs:jsr305:jar:3.0.2:compile
[INFO] |  +- com.google.code.findbugs:annotations:jar:3.0.1:compile
[INFO] |  |  +- net.jcip:jcip-annotations:jar:1.0:compile
[INFO] |  |  \- (com.google.code.findbugs:jsr305:jar:3.0.1:compile - omitted for conflict with 3.0.2)
[INFO] |  +- (org.apache.commons:commons-lang3:jar:3.12.0:compile - omitted for duplicate)
[INFO] |  +- com.google.code.gson:gson:jar:2.9.0:compile
[INFO] |  +- javax.measure:jsr-275:jar:0.9.1:compile
[INFO] |  +- com.google.inject.extensions:guice-multibindings:jar:4.2.3:compile
[INFO] |  |  \- (com.google.inject:guice:jar:4.2.3:compile - omitted for conflict with 5.1.0)
[INFO] |  +- commons-io:commons-io:jar:2.11.0:compile
[INFO] |  +- org.apache.httpcomponents:httpasyncclient:jar:4.1.5:compile
[INFO] |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.15:compile
[INFO] |  |  +- org.apache.httpcomponents:httpcore-nio:jar:4.4.15:compile
[INFO] |  |  |  \- (org.apache.httpcomponents:httpcore:jar:4.4.15:compile - omitted for duplicate)
[INFO] |  |  +- (org.apache.httpcomponents:httpclient:jar:4.5.13:compile - omitted for duplicate)
[INFO] |  |  \- (commons-logging:commons-logging:jar:1.2:compile - omitted for conflict with 1.1.3)
[INFO] |  +- org.apache.logging.log4j:log4j-core:jar:2.17.1:compile
[INFO] |  |  \- (org.apache.logging.log4j:log4j-api:jar:2.17.1:compile - omitted for duplicate)
[INFO] |  \- org.apache.logging.log4j:log4j-api:jar:2.17.1:compile
[INFO] +- com.amazonaws:amazon-kinesis-client:jar:1.14.8:compile
[INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
[INFO] |  +- (com.amazonaws:aws-java-sdk-dynamodb:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
[INFO] |  +- (com.amazonaws:aws-java-sdk-kinesis:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
[INFO] |  +- com.amazonaws:aws-java-sdk-cloudwatch:jar:1.12.162:compile (version managed from 1.12.130)
[INFO] |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
[INFO] |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
[INFO] |  +- com.google.guava:guava:jar:31.0.1-jre:compile
[INFO] |  |  +- com.google.guava:failureaccess:jar:1.0.1:compile
[INFO] |  |  +- com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava:compile
[INFO] |  |  +- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
[INFO] |  |  +- org.checkerframework:checker-qual:jar:3.12.0:compile
[INFO] |  |  +- com.google.errorprone:error_prone_annotations:jar:2.7.1:compile
[INFO] |  |  \- com.google.j2objc:j2objc-annotations:jar:1.3:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:3.19.4:compile
[INFO] |  +- (org.apache.commons:commons-lang3:jar:3.12.0:compile - omitted for duplicate)
[INFO] |  +- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.13.0:compile
[INFO] +- com.amazonaws:aws-lambda-java-core:jar:1.2.1:compile
[INFO] +- com.amazonaws:aws-lambda-java-events:jar:2.2.9:compile
[INFO] |  \- joda-time:joda-time:jar:2.6:compile
[INFO] +- org.jcodec:jcodec:jar:0.2.3:compile
[INFO] |  \- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
[INFO] +- org.jcodec:jcodec-javase:jar:0.2.3:compile
[INFO] |  \- (org.jcodec:jcodec:jar:0.2.3:compile - omitted for duplicate)
[INFO] +- junit:junit:jar:4.13.2:test
[INFO] |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO] +- org.projectlombok:lombok:jar:1.18.22:provided
[INFO] +- org.apache.commons:commons-lang3:jar:3.12.0:compile
[INFO] +- com.amazonaws:aws-lambda-java-log4j2:jar:1.5.1:compile
[INFO] |  +- (com.amazonaws:aws-lambda-java-core:jar:1.2.1:compile - omitted for duplicate)
[INFO] |  +- (org.apache.logging.log4j:log4j-core:jar:2.17.1:compile - omitted for duplicate)
[INFO] |  \- (org.apache.logging.log4j:log4j-api:jar:2.17.1:compile - omitted for duplicate)
[INFO] +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.17.2:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.25:compile
[INFO] |  +- (org.apache.logging.log4j:log4j-api:jar:2.17.2:compile - omitted for conflict with 2.17.1)
[INFO] |  \- (org.apache.logging.log4j:log4j-core:jar:2.17.2:runtime - omitted for conflict with 2.17.1)
[INFO] \- org.powermock:powermock-mockito-release-full:pom:1.6.4:test
[INFO]    +- org.powermock:powermock-core:jar:1.6.4:test
[INFO]    |  +- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- org.javassist:javassist:jar:3.20.0-GA:test
[INFO]    +- org.powermock:powermock-api-mockito:jar:1.6.4:test
[INFO]    |  +- org.mockito:mockito-core:jar:1.10.19:test
[INFO]    |  |  +- (org.hamcrest:hamcrest-core:jar:1.1:test - omitted for conflict with 1.3)
[INFO]    |  |  \- (org.objenesis:objenesis:jar:2.1:test - omitted for duplicate)
[INFO]    |  +- (org.hamcrest:hamcrest-core:jar:1.3:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4-common:jar:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.4:test - omitted for conflict with 4.13.2)
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4:jar:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.12:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4-legacy:jar:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.3:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit3:jar:1.6.4:test
[INFO]    |  +- (junit:junit:jar:3.8.1:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-reflect:jar:1.6.4:test
[INFO]    |  \- org.objenesis:objenesis:jar:2.1:test
[INFO]    +- org.powermock:powermock-api-support:jar:1.6.4:test
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-testng:jar:1.6.4:test
[INFO]    |  +- org.testng:testng:jar:6.8.21:test
[INFO]    |  |  +- org.beanshell:bsh:jar:2.0b4:test
[INFO]    |  |  \- com.beust:jcommander:jar:1.27:test
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-module-testng-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-testng-common:jar:1.6.4:test
[INFO]    |  +- (org.testng:testng:jar:6.8.21:test - omitted for duplicate)
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-classloading-base:jar:1.6.4:test
[INFO]    |  +- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-core:jar:sources:1.6.4:test
[INFO]    |  +- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.javassist:javassist:jar:3.20.0-GA:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-api-mockito:jar:sources:1.6.4:test
[INFO]    |  +- (org.mockito:mockito-core:jar:1.10.19:test - omitted for duplicate)
[INFO]    |  +- (org.hamcrest:hamcrest-core:jar:1.3:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4-common:jar:sources:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.4:test - omitted for conflict with 4.13.2)
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4:jar:sources:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.12:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit4-legacy:jar:sources:1.6.4:test
[INFO]    |  +- (junit:junit:jar:4.3:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-junit3:jar:sources:1.6.4:test
[INFO]    |  +- (junit:junit:jar:3.8.1:test - omitted for conflict with 4.13.2)
[INFO]    |  \- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-module-testng:jar:sources:1.6.4:test
[INFO]    |  +- (org.testng:testng:jar:6.8.21:test - omitted for duplicate)
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-module-testng-common:jar:1.6.4:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-reflect:jar:sources:1.6.4:test
[INFO]    |  \- (org.objenesis:objenesis:jar:2.1:test - omitted for duplicate)
[INFO]    +- org.powermock:powermock-api-support:jar:sources:1.6.4:test
[INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
[INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
[INFO]    \- org.powermock:powermock-classloading-base:jar:sources:1.6.4:test
[INFO]       +- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
[INFO]       \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
```

bdhandap commented:
> ```
> [INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ amazon-kinesis-video-streams-parser-library ---
> [WARNING] The artifact xml-apis:xml-apis:jar:2.0.2 has been relocated to xml-apis:xml-apis:jar:1.0.b2
> [INFO] com.amazonaws:amazon-kinesis-video-streams-parser-library:jar:1.2.5
> [INFO] +- com.amazonaws:aws-java-sdk-rekognition:jar:1.12.162:compile
> [INFO] |  +- com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile
> [INFO] |  |  +- (commons-logging:commons-logging:jar:1.1.3:compile - omitted for conflict with 1.2)
> [INFO] |  |  +- commons-codec:commons-codec:jar:1.15:compile
> [INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.13:compile
> [INFO] |  |  |  +- (org.apache.httpcomponents:httpcore:jar:4.4.13:compile - omitted for conflict with 4.4.15)
> [INFO] |  |  |  +- (commons-logging:commons-logging:jar:1.2:compile - omitted for conflict with 1.1.3)
> [INFO] |  |  |  \- (commons-codec:commons-codec:jar:1.11:compile - omitted for conflict with 1.15)
> [INFO] |  |  +- software.amazon.ion:ion-java:jar:1.0.2:compile
> [INFO] |  |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile (version managed from 2.12.6)
> [INFO] |  |  |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.12.3:compile
> [INFO] |  |  |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.12.3:compile - omitted for conflict with 2.13.0)
> [INFO] |  |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.12.3:compile (version managed from 2.12.6)
> [INFO] |  |  |  +- (com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile - version managed from 2.12.6; omitted for duplicate)
> [INFO] |  |  |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.12.3:compile - omitted for duplicate)
> [INFO] |  |  \- (joda-time:joda-time:jar:2.8.1:compile - omitted for conflict with 2.6)
> [INFO] |  \- com.amazonaws:jmespath-java:jar:1.12.162:compile
> [INFO] |     \- (com.fasterxml.jackson.core:jackson-databind:jar:2.12.3:compile - version managed from 2.12.6; omitted for duplicate)
> [INFO] +- com.amazonaws:aws-java-sdk-kinesisvideo:jar:1.12.162:compile
> [INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  +- io.netty:netty-codec-http:jar:4.1.72.Final:compile
> [INFO] |  |  +- io.netty:netty-common:jar:4.1.72.Final:compile
> [INFO] |  |  +- io.netty:netty-buffer:jar:4.1.72.Final:compile
> [INFO] |  |  |  \- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- io.netty:netty-transport:jar:4.1.72.Final:compile
> [INFO] |  |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  |  \- (io.netty:netty-resolver:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- io.netty:netty-codec:jar:4.1.72.Final:compile
> [INFO] |  |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  |  \- (io.netty:netty-transport:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  \- (io.netty:netty-handler:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  +- io.netty:netty-handler:jar:4.1.72.Final:compile
> [INFO] |  |  +- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- io.netty:netty-resolver:jar:4.1.72.Final:compile
> [INFO] |  |  |  \- (io.netty:netty-common:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- (io.netty:netty-buffer:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- (io.netty:netty-transport:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  +- (io.netty:netty-codec:jar:4.1.72.Final:compile - omitted for duplicate)
> [INFO] |  |  \- io.netty:netty-tcnative-classes:jar:2.0.46.Final:compile
> [INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] +- com.amazonaws:aws-java-sdk-kinesis:jar:1.12.162:compile
> [INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] +- com.amazonaws:aws-java-sdk-dynamodb:jar:1.12.162:compile
> [INFO] |  +- com.amazonaws:aws-java-sdk-s3:jar:1.12.162:compile
> [INFO] |  |  +- com.amazonaws:aws-java-sdk-kms:jar:1.12.162:compile
> [INFO] |  |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] +- com.amazonaws:amazon-kinesis-video-streams-producer-sdk-java:jar:1.12.0:compile
> [INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.11.602; omitted for duplicate)
> [INFO] |  +- (com.amazonaws:aws-java-sdk-kinesisvideo:jar:1.12.162:compile - version managed from 1.11.602; omitted for duplicate)
> [INFO] |  +- com.google.inject:guice:jar:5.1.0:compile
> [INFO] |  |  +- javax.inject:javax.inject:jar:1:compile
> [INFO] |  |  +- aopalliance:aopalliance:jar:1.0:compile
> [INFO] |  |  \- (com.google.guava:guava:jar:30.1-jre:compile - omitted for conflict with 31.0.1-jre)
> [INFO] |  +- com.google.code.findbugs:jsr305:jar:3.0.2:compile
> [INFO] |  +- com.google.code.findbugs:annotations:jar:3.0.1:compile
> [INFO] |  |  +- net.jcip:jcip-annotations:jar:1.0:compile
> [INFO] |  |  \- (com.google.code.findbugs:jsr305:jar:3.0.1:compile - omitted for conflict with 3.0.2)
> [INFO] |  +- (org.apache.commons:commons-lang3:jar:3.12.0:compile - omitted for duplicate)
> [INFO] |  +- com.google.code.gson:gson:jar:2.9.0:compile
> [INFO] |  +- javax.measure:jsr-275:jar:0.9.1:compile
> [INFO] |  +- com.google.inject.extensions:guice-multibindings:jar:4.2.3:compile
> [INFO] |  |  \- (com.google.inject:guice:jar:4.2.3:compile - omitted for conflict with 5.1.0)
> [INFO] |  +- commons-io:commons-io:jar:2.11.0:compile
> [INFO] |  +- org.apache.httpcomponents:httpasyncclient:jar:4.1.5:compile
> [INFO] |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.15:compile
> [INFO] |  |  +- org.apache.httpcomponents:httpcore-nio:jar:4.4.15:compile
> [INFO] |  |  |  \- (org.apache.httpcomponents:httpcore:jar:4.4.15:compile - omitted for duplicate)
> [INFO] |  |  +- (org.apache.httpcomponents:httpclient:jar:4.5.13:compile - omitted for duplicate)
> [INFO] |  |  \- (commons-logging:commons-logging:jar:1.2:compile - omitted for conflict with 1.1.3)
> [INFO] |  +- org.apache.logging.log4j:log4j-core:jar:2.17.1:compile
> [INFO] |  |  \- (org.apache.logging.log4j:log4j-api:jar:2.17.1:compile - omitted for duplicate)
> [INFO] |  \- org.apache.logging.log4j:log4j-api:jar:2.17.1:compile
> [INFO] +- com.amazonaws:amazon-kinesis-client:jar:1.14.8:compile
> [INFO] |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
> [INFO] |  +- (com.amazonaws:aws-java-sdk-dynamodb:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
> [INFO] |  +- (com.amazonaws:aws-java-sdk-kinesis:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
> [INFO] |  +- com.amazonaws:aws-java-sdk-cloudwatch:jar:1.12.162:compile (version managed from 1.12.130)
> [INFO] |  |  +- (com.amazonaws:aws-java-sdk-core:jar:1.12.162:compile - version managed from 1.12.130; omitted for duplicate)
> [INFO] |  |  \- (com.amazonaws:jmespath-java:jar:1.12.162:compile - omitted for duplicate)
> [INFO] |  +- com.google.guava:guava:jar:31.0.1-jre:compile
> [INFO] |  |  +- com.google.guava:failureaccess:jar:1.0.1:compile
> [INFO] |  |  +- com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava:compile
> [INFO] |  |  +- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
> [INFO] |  |  +- org.checkerframework:checker-qual:jar:3.12.0:compile
> [INFO] |  |  +- com.google.errorprone:error_prone_annotations:jar:2.7.1:compile
> [INFO] |  |  \- com.google.j2objc:j2objc-annotations:jar:1.3:compile
> [INFO] |  +- com.google.protobuf:protobuf-java:jar:3.19.4:compile
> [INFO] |  +- (org.apache.commons:commons-lang3:jar:3.12.0:compile - omitted for duplicate)
> [INFO] |  +- commons-logging:commons-logging:jar:1.2:compile
> [INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.13.0:compile
> [INFO] +- com.amazonaws:aws-lambda-java-core:jar:1.2.1:compile
> [INFO] +- com.amazonaws:aws-lambda-java-events:jar:2.2.9:compile
> [INFO] |  \- joda-time:joda-time:jar:2.6:compile
> [INFO] +- org.jcodec:jcodec:jar:0.2.3:compile
> [INFO] |  \- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
> [INFO] +- org.jcodec:jcodec-javase:jar:0.2.3:compile
> [INFO] |  \- (org.jcodec:jcodec:jar:0.2.3:compile - omitted for duplicate)
> [INFO] +- junit:junit:jar:4.13.2:test
> [INFO] |  \- org.hamcrest:hamcrest-core:jar:1.3:test
> [INFO] +- org.projectlombok:lombok:jar:1.18.22:provided
> [INFO] +- org.apache.commons:commons-lang3:jar:3.12.0:compile
> [INFO] +- com.amazonaws:aws-lambda-java-log4j2:jar:1.5.1:compile
> [INFO] |  +- (com.amazonaws:aws-lambda-java-core:jar:1.2.1:compile - omitted for duplicate)
> [INFO] |  +- (org.apache.logging.log4j:log4j-core:jar:2.17.1:compile - omitted for duplicate)
> [INFO] |  \- (org.apache.logging.log4j:log4j-api:jar:2.17.1:compile - omitted for duplicate)
> [INFO] +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.17.2:compile
> [INFO] |  +- org.slf4j:slf4j-api:jar:1.7.25:compile
> [INFO] |  +- (org.apache.logging.log4j:log4j-api:jar:2.17.2:compile - omitted for conflict with 2.17.1)
> [INFO] |  \- (org.apache.logging.log4j:log4j-core:jar:2.17.2:runtime - omitted for conflict with 2.17.1)
> [INFO] \- org.powermock:powermock-mockito-release-full:pom:1.6.4:test
> [INFO]    +- org.powermock:powermock-core:jar:1.6.4:test
> [INFO]    |  +- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- org.javassist:javassist:jar:3.20.0-GA:test
> [INFO]    +- org.powermock:powermock-api-mockito:jar:1.6.4:test
> [INFO]    |  +- org.mockito:mockito-core:jar:1.10.19:test
> [INFO]    |  |  +- (org.hamcrest:hamcrest-core:jar:1.1:test - omitted for conflict with 1.3)
> [INFO]    |  |  \- (org.objenesis:objenesis:jar:2.1:test - omitted for duplicate)
> [INFO]    |  +- (org.hamcrest:hamcrest-core:jar:1.3:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4-common:jar:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.4:test - omitted for conflict with 4.13.2)
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4:jar:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.12:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4-legacy:jar:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.3:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit3:jar:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:3.8.1:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-reflect:jar:1.6.4:test
> [INFO]    |  \- org.objenesis:objenesis:jar:2.1:test
> [INFO]    +- org.powermock:powermock-api-support:jar:1.6.4:test
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-testng:jar:1.6.4:test
> [INFO]    |  +- org.testng:testng:jar:6.8.21:test
> [INFO]    |  |  +- org.beanshell:bsh:jar:2.0b4:test
> [INFO]    |  |  \- com.beust:jcommander:jar:1.27:test
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-module-testng-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-testng-common:jar:1.6.4:test
> [INFO]    |  +- (org.testng:testng:jar:6.8.21:test - omitted for duplicate)
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-classloading-base:jar:1.6.4:test
> [INFO]    |  +- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-core:jar:sources:1.6.4:test
> [INFO]    |  +- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.javassist:javassist:jar:3.20.0-GA:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-api-mockito:jar:sources:1.6.4:test
> [INFO]    |  +- (org.mockito:mockito-core:jar:1.10.19:test - omitted for duplicate)
> [INFO]    |  +- (org.hamcrest:hamcrest-core:jar:1.3:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4-common:jar:sources:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.4:test - omitted for conflict with 4.13.2)
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4:jar:sources:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.12:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit4-legacy:jar:sources:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:4.3:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-module-junit4-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-junit3:jar:sources:1.6.4:test
> [INFO]    |  +- (junit:junit:jar:3.8.1:test - omitted for conflict with 4.13.2)
> [INFO]    |  \- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-module-testng:jar:sources:1.6.4:test
> [INFO]    |  +- (org.testng:testng:jar:6.8.21:test - omitted for duplicate)
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-module-testng-common:jar:1.6.4:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-reflect:jar:sources:1.6.4:test
> [INFO]    |  \- (org.objenesis:objenesis:jar:2.1:test - omitted for duplicate)
> [INFO]    +- org.powermock:powermock-api-support:jar:sources:1.6.4:test
> [INFO]    |  +- (org.powermock:powermock-core:jar:1.6.4:test - omitted for duplicate)
> [INFO]    |  \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> [INFO]    \- org.powermock:powermock-classloading-base:jar:sources:1.6.4:test
> [INFO]       +- (org.powermock:powermock-api-support:jar:1.6.4:test - omitted for duplicate)
> [INFO]       \- (org.powermock:powermock-reflect:jar:1.6.4:test - omitted for duplicate)
> ```

I tested the lambda code with uber jar containing duplicate classes and it's able to generated bounding boxes in derived KVS streams. Works fine.
